charts/mod-dcb/Chart.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+name: mod-dcb
+description: A Helm chart Folio mod-circulation for Kubernetes
```

Comment:
Please review and fix description in newely created charts

Reply:
Done